internal/config/config.go

```diff
@@ -88,17 +88,18 @@ type TagFilter struct {
 type RunningOutput struct {
 	Name   string
 	Output outputs.Output
+	Config *PrefixFilterConfig
 }
 
 type RunningPlugin struct {
 	Name   string
 	Plugin plugins.Plugin
-	Config *PluginConfig
+	Config *PrefixFilterConfig
 }
 
-// PluginConfig containing a name, interval, and drop/pass prefix lists
+// PrefixFilterConfig contains the name, interval (for plugin) and drop/pass prefix lists
 // Also lists the tags to filter
-type PluginConfig struct {
+type PrefixFilterConfig struct {
```

Comment:
I don't like overloading this, I would prefer if we kept `PluginConfig` and add another struct called just `Filter` that can be part of PluginConfig. Then probably will need to add an OutputConfig struct